src/TeamSpeak3QueryApi/TeamSpeakClient.cs

```diff
@@ -15,6 +15,7 @@
 using TeamSpeak3QueryApi.Net.Query.Parameters;
 using TeamSpeak3QueryApi.Net.Query.Responses;
 using TeamSpeak3QueryApi.Net.Query.Enums;
+using TeamSpeak3QueryApi.Net.Query.Protocols;
```

Comment:
While I think namespaces like `Enums` etc. might be good for _applications_, I don't think that a library should use that pattern. For example `System.Windows.Forms` doesn't have something like `Controls`, `Components` etc. either. This way, the core principle mentioned above is easier to follow.

Reply:
I think this is a matter of taste. I would like to stay like this. It´s for me better readable if we stay in folder and not change the classnames. Or we could change the classnames and stay also in the folders?